package.json

```diff
@@ -0,0 +1,37 @@
+{
+  "name": "devx-ts-starter",
+  "version": "0.0.1",
+  "description": "Starter template for TypeScript projects.",
+  "private": true,
```

Comment:
Note: The idea here is to always start off with a private package as a default. Since this starter template comes with a full release pipeline, this flag will avoid unintentional publication of libs that were never meant to be in a package repo.